### DISCUSSION TITLE
ChatList

### DISCUSSION THREAD

**Discussion by hirbod** [flyerhq/react-native-chat-ui#41] [Ideas / Feature Requests] [answered]
I love this library. A lot cleaner than gifted-chat. I would love to see a ChatList / Message List as a Feature.
It should handle like avatar, last message, unseen count etc.

I know, this can easily be done by myself (and I am already building it), but I feel like this library already brings a quality themed chat, so a list screen is the last missing piece.

**Comment by demchenkoalex** [comment, marked as answer]
Hey @Hirbod. I don't remember where I replied to the same request (probably in the Flutter repo) but TL;DR the chat list screen can look different for everyone and tbh creating a list with an avatar is like 5 minutes of work for a dev. So when you're creating one for yourself you spend 5 minutes, but if I build something "generic" I will get tons of requests that something's missing. So I deliberately skipped the chat list part and don't plan to include it in the repo in the near future.

**Reply by hirbod** [reply]
Hi @demchenkoalex, thats a fair and understandable answer. As said, I already built one myself and it is indeed not a big deal. :)
